change/@fluentui-react-native-experimental-switch-c30f2890-78de-40b6-8126-6eeb6d700039.json

```diff
@@ -1,7 +0,0 @@
-{
-  "type": "minor",
-  "comment": "Added Switch control for win32",
```

Comment:
Intentional?

Reply:
Yes, beachbball has been complaining about it so I deleted it. Probably shouldn't be part of this PR, but oh well.